tests/keras/test_callbacks.py

```diff
@@ -547,7 +547,7 @@ def callbacks_factory(histogram_freq, embeddings_freq=1):
 @keras_test
 @pytest.mark.skipif((K.backend() != 'tensorflow'),
                     reason='Requires TensorFlow backend')
-def test_TensorBoard_histogram_freq_must_have_validation_data(tmpdir):
+def pipq(tmpdir):
```

Comment:
Why rename this test? If it isn't prefixed by "test", it will not be run by pytest.

Reply:
Typo, fixed